frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnFieldFooter.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { Button } from 'patternfly-react';
+
+export interface MultiColumnFieldHeader {
+  addLabel: string;
+  onAdd: () => void;
+}
+
+const MultiColumnFieldFooter: React.FC<MultiColumnFieldHeader> = ({ addLabel, onAdd }) => (
+  <Button type="button" bsStyle="link" className="btn-link--no-btn-default-values" onClick={onAdd}>
+    <PlusCircleIcon className="pairs-list__add-icon" />
```

Comment:
where does `pairs-list__add-icon` come from?

Reply:
Usage of `pairs-list__add-icon` at least requires a comment because it's being used to align with existing styling. Note that it comes from `_name-value-editor.scss` which is a poor dependency for this file and should be documented.

Reply:
Just saw that this will be removed by PR: https://github.com/openshift/console/pull/2038/files#diff-06f41512c4e8bead92c3b726327ebf21L64

Reply:
Removed usage of that class.

Reply:
This will go away when we move to PF4. As the spacing is using just a blank space (see pr referenced above)